core/internal/features_test.go

```diff
@@ -1446,17 +1463,26 @@ p2pBootstrapPeers  = [
 ]
 keyBundleID        = "%s"
 transmitterAddress = "%s"
-observationTimeout = "20s"
+observationTimeout = "400ms"
```

Comment:
would it make sense to also set the gracePeriod here so as to avoid defaults changing later invalidating the test?

Reply:
its set here https://github.com/smartcontractkit/chainlink/pull/3980/files#diff-ce02a859bd593cfb7f12803e10675dfc2501631a8a8d59b5567aac4570f4e2adR1373 - currently only a global env var, not a per job var